src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -280,16 +280,14 @@ def initialize_newborns(self, pop_data: SimulantData) -> None:
         new_births["ssn"] = self.ssn_generator.generate(new_births).ssn
         # Check for SSN duplicates with existing SSNs
         ssn_values = ssns.loc[ssns != ""]
-        all_ssns = pd.concat([ssn_values, new_births.ssn])
-        duplicate_mask = all_ssns.duplicated()
+        duplicate_mask = new_births["ssn"].isin(ssn_values) | new_births.duplicated(subset=["ssn"])
```

Comment:
`new_births` can't have duplicates due to the way the ssn generator works unless I misunderstood that method (in which case the initialization method is not correct).